pkg/resource/model_package_group/hooks.go

```diff
@@ -29,46 +30,32 @@ var (
 		ackrequeue.DefaultRequeueAfterDuration,
 	)
 )
+var resourceName = "Model package group"
```

Comment:
It is global because across the deleting and creating feature group pr's (#51 and #52), this variable is reused in two different functions in this hooks.go file: modifyingStatuses() and requeueUntilCanModify().